daemon/execdriver/native/exec.go

```diff
@@ -52,7 +52,7 @@ func (d *Driver) Exec(c *execdriver.Command, processConfig *execdriver.ProcessCo
 			p.Wait()
 			return -1, err
 		}
-		startCallback(&c.ProcessConfig, pid)
+		startCallback(&c.ProcessConfig, pid, nil)
```

Comment:
You could create and immediately close the channel to pass here as we're doing  inhttps://github.com/docker/docker/blob/master/daemon/execdriver/native/driver.go#L205